src/Hill.jl

```diff
@@ -43,33 +43,53 @@ function grad_helper!(G, f, range, params)
 end
 
 import LineSearches
+function optimize_hill(conc::Vector, g1::Matrix, g2::Matrix, initial; maxstep = 100000)
 
-
-function optimize_helper(f, g!, low::Vector, high::Vector, maxstep::Int)
-    initial_x = low + (high - low) / 3.0
+    f(x) = residHill(x, conc, g1, g2)
+    g!(x, G) = grad_helper!(x, G, f, 10:13)
 
     method = Fminbox(LBFGS(linesearch = LineSearches.BackTracking()))
 
     options = Optim.Options(outer_iterations = 2, show_trace = true, iterations = maxstep)
     results = optimize(f, g!, low, high, initial_x, method, options)
 
-    println(results)
+    low = [minimum(conc), 1e-9, 1e-9, 0.1, 1e-9, 1e-9, 0.0, 0.0, 0.25, 3, 5, 0, 0]
+    high = [maximum(conc), 1.0, 1.0, 10.0, 1.0, 1.0, 3.0, 3.0, 0.75, 50, 50, 50, 50]
 
-    return Optim.minimum(results), Optim.minimizer(results)
+    return optimize_helper(f, g!, low, high, initial, maxstep)
 end
 
+function _keep_lowest(xs, N)
+    @argcheck 1 ≤ N ≤ 20
+    partialsort(xs, 1:N, by = p -> p.value)
+end
 
-""" Hill optimization function. """
-function optimize_hill(conc::Vector, g1::Matrix, g2::Matrix; maxstep = 100000)
-    f(x) = residHill(x, conc, g1, g2)
-    g!(G, x) = grad_helper!(G, f, 10:13, x)
+function sobol_starting_points(minimization_problem::MinimizationProblem, N::Integer)
+    @unpack objective, lower_bounds, upper_bounds = minimization_problem
+    s = SobolSeq(lower_bounds, upper_bounds)
+    skip(s, N)                  # better uniformity
+    map(fetch, map(x -> @spawn(LocationValue(x, objective(x))), Iterators.take(s, N)))
+end
```

Comment:
@aarmey is it possible to write this line without using `LocationValue`?
 I deleted this struct (the main reason was I wasn't sure what exactly it is doing). Now, I have this error `no method matching length(::LocationValue{Array{Float64,1},Float64})` which goes back to this.

Reply:
Not sure what this is doing either. Maybe you just need to put `MultistartOptimization.LocationValue` because it's not exported?

Reply:
Do you have to copy this function out of the package too?

Reply:
I tried just calling it by doing `MultistartOptimization.sobol_starting_points`, but the error occurred. I thought if I copy this function and change that specific line, the error would fix.